framework/source/class/qx/io/request/Xhr.js

```diff
@@ -254,7 +255,7 @@ qx.Class.define("qx.io.request.Xhr",
       }
 
       // By default, set content-type urlencoded for requests with body
-      if (this.getRequestData() && isAllowsBody) {
+      if (this.getRequestData() && isAllowsBody && !isFormData) {
```

Comment:
Is there necessarily a one-to-one relationship between the requestData class being FormData, and `headers["Content-Type"] being set already? Would it make more sense, here, to test whether `! ("Content-Type" in headers)` ?

Reply:
For FormData to work as a payload, Content-Type must not be set. The way the code presently works, qx will always set the Content-Type, it can be overwritten by th user, but there is no way to NOT set it, and unfortunately this is the only way FormData Posts will work.

Reply:
Ii think you misunderstood me. I'm not saying that a change isn't required. I agree, one is needed. What I'm suggesting is that this fix might not be as general as we'd like. If someone manually added that header, i.e., if `headers["Content-Type"] already has a value at this point for any reason, isn't that a better test than just whether the content type is FormData?

Reply:
Or maybe I misunderstand the issue. "Content-Type must not be set" at all if the content type if FormData? That sounds odd...

Reply:
Disregard my comments. I understand what's happening now. All good.